### DISCUSSION TITLE
Just curious: what are your use cases for this package?

### DISCUSSION THREAD

**Discussion by igorcafe** [kljohann/mpv.el#39] [Show and tell]

**Comment by kljohann** [comment]
Hi @igorcafe, I initially created this package to implement timestamped links to videos /media files when transcribing lectures etc. using Org mode.  See the [configuration snippets in the wiki](https://github.com/kljohann/mpv.el/wiki).  But it can be used as a general purpose way to remote-control mpv from Emacs, with its main current limitation being that multiple concurrent mpv processes aren't supported.  And there are some Emacs packages that depend on it for that purpose.  In the meantime there are also other Emacs packages that (re-)implement similar functionality, so everyone can choose what works best for them. :slightly_smiling_face:
Maybe others want to chime in with their use cases here.

**Reply by igorcafe** [reply]
Nice.
I started using yeetube.el to watch youtube videos and I make notes in org, so it would be very useful to make timestamp based notes, and I found out that org-mpv-notes is exactly for that
